### PR TITLE
Fixes #9339 document log workaround in IDEs

### LINKED ISSUE #9339: Running JUnit tests in IDE doesn't set the LogManager
**Describe the bug**
When running a QuarkustTest class as JUnit in Eclipse IDE, the quarkus.log.* entries set in application.properties are ignored, and logging defaults are used.

**Expected behavior**
I expect to be able to see the same configured logging settings in a Maven run as in a IDE testclass run.

**Actual behavior**
Rightclick a QuarkusTest class and select to run as JUnit. None of your debug() statements will write something in the console because the default log level is INFO and the quarkus.log.level=DEBUG from application.properties is ignored. The file is properly parsed, just the logging not initialized.

**To Reproduce**
- Take any quickstart project, 
- add quarkus.log.level=DEBUG in its application.properties
- add a debug() statement in one of the unit tests
- run the maven install, the debug statement writes to the output
- run the class as junit, the debug statement writes nothing

**Configuration**
```
quarkus.log.level=DEBUG 
```

**Screenshots**
(If applicable, add screenshots to help explain your problem.)

**Environment (please complete the following information):**
 - Output of `uname -a` or `ver`: Windows 10
 - Output of `java -version`: 11 (probably not relevant)
 - GraalVM version (if different from Java): 
 - Quarkus version or git rev: 1.4.2
 - Build tool (ie. output of `mvnw --version` or `gradlew --version`): Maven 3.6.3, Eclipse 2020-03

**Additional context**
(Add any other context about the problem here.)
See also Zulip chat https://quarkusio.zulipchat.com/#narrow/stream/187030-users/topic/absolutebeginner

### PR DESCRIPTION
**Pull request by sorin-costea**
Fixes #9339 as discussed, mentions the ways of adding the workaround JVM LogManager argument to Eclipse, VSCode and IntelliJ

### CODE REVIEW

**Comment by famod** [comment]
I think I'll extend this with `${maven.home}` in #9523.

**Comment by famod** [comment]
Btw, @sorin-costea if you add "Fixes #9339" to your message then it will link the issuse and auto-close it once merged.

**Comment by sorin-costea** [comment]
I added it to the title oops

File: docs/src/main/asciidoc/logging.adoc [hunk 1]
```diff
@@ -249,6 +247,33 @@ test {
...
+        "vmargs": [ "-Djava.util.logging.manager=org.jboss.logmanager.LogManager ..." ],
+        ...
+    },
+  ...
+]
+----
+
+=== IntelliJ JUnit template
+
+In the same screen where you edit your run configuration you can edit the JUnit template to add the workaround Java VM argument `-Djava.util.logging.manager=org.jboss.logmanager.LogManager`.
```
Comment (famod) [inline docs/src/main/asciidoc/logging.adoc:276]: I am not an IntelliJ user but the Maven integration should automatically pick up `systemPropertyVariables` from the surefire config in pom.xml.

Comment (sorin-costea) [reply docs/src/main/asciidoc/logging.adoc:276]: I took that from the issue discussion, as I'm not an IntelliJ user either. Just Eclipse and VSCode are tested.

Comment (rquinio) [reply docs/src/main/asciidoc/logging.adoc:276]: I can confirm surefire systemPropertyVariables get picked-up by https://github.com/JetBrains/intellij-community/blob/master/plugins/maven/src/main/java/org/jetbrains/idea/maven/execution/MavenJUnitPatcher.java#L162 , so it should work out of the box.

**Comment by sorin-costea** [comment]
Okay, updated the IntelliJ section

**Comment by famod** [comment]
Can somebody review/merge this please? @stuartwdouglas @gsmet @geoand maybe?
I am waiting for it in #9523. Thanks!

**Review by stuartwdouglas — APPROVED** [review]

**Pull request merged**